staging/src/k8s.io/apiserver/pkg/server/filters/reqmgmt.go

```diff
@@ -159,84 +396,192 @@ func (tr *TypedConfigObjectReference) String() string {
+		// hashValue := reqMgmt.hashFlowID(fs.Name, flowDistinguisher)
+		hashValue := hashFlowID(fs.Name, flowDistinguisher)
+
+		// TODO(aaron-prindle) verify this is needed?
+		rmState.updateACV(ps) // TODO(aaron-prindle) rename updateACV?
+		ps.fqs.SetConcurrencyLimit(ps.concurrencyLimit)
+
+		// attempt to enqueue and dispatch the package
+		// TODO(aaron-prindle) use PriorityLevelState concurrencylimit OR
+		// fqs concurrency
+
 		execute, afterExecute := ps.fqs.Wait(hashValue, *ps.config.HandSize)
+
 		if execute {
 			klog.V(5).Infof("Serving %v after queuing\n", r)
+
 			handler.ServeHTTP(w, r)
+
 			afterExecute()
 		} else {
 			klog.V(5).Infof("Rejecting %v\n", r)
-			tooManyRequests(r, w)
+			// tooManyRequests(r, w) // TODO(aaron-prindle)
 		}
-
 		return
 	})
 }
 
-func (requestManagement) computeFlowDistinguisher(r *http.Request, fs *rmtypesv1a1.FlowSchema) string {
-	// TODO: implement
-	return ""
+func hash(s string) uint64 {
+	h := fnv.New64a()
+	h.Write([]byte(s))
+	return h.Sum64()
 }
 
-func (requestManagement) hashFlowID(fsName, fDistinguisher string) uint64 {
+// func (requestManagement) computeFlowDistinguisher(r *http.Request, fs *rmtypesv1a1.FlowSchema) string {
+// 	// TODO: implement
+// 	// TODO(aaron-prindle) CHANGE replace w/ proper implementation
+// 	return fs.Name
+// }
+
+// func (requestManagement) hashFlowID(fsName, fDistinguisher string) uint64 {
+// 	// TODO: implement
+// 	// TODO(aaron-prindle) verify implementation
+// 	return hash(fmt.Sprintf("%s,%s", fsName, fDistinguisher))
+// }
+
+// func (requestManagement) pickFlowSchema(r *http.Request, flowSchemas FlowSchemaSeq, priorityLevelStates map[string]*PriorityLevelState) *rmtypesv1a1.FlowSchema {
+// 	// TODO(aaron-prindle) CHANGE replace w/ proper implementation
+// 	priority := r.Header.Get("PRIORITY")
+// 	idx, err := strconv.Atoi(priority)
+// 	if err != nil {
+// 		panic("strconv.Atoi(priority) errored")
+// 	}
+// 	// TODO(aaron-prindle) can also use MatchingPrecedence for dummy method
+// 	return flowSchemas[idx]
+// }
+
+// func (requestManagement) requestPriorityState(r *http.Request, fs *rmtypesv1a1.FlowSchema, priorityLevelStates map[string]*PriorityLevelState) *PriorityLevelState {
+// 	// TODO: implement
+// 	out, ok := priorityLevelStates[fs.Spec.PriorityLevelConfiguration.Name]
+// 	if !ok {
+// 		panic("NOT OK!!")
+// 	}
+// 	return out
+// }
+
+func computeFlowDistinguisher(r *http.Request, fs *rmtypesv1a1.FlowSchema) string {
 	// TODO: implement
-	return 0
+	// TODO(aaron-prindle) CHANGE replace w/ proper implementation
+	return fs.Name
 }
 
-func (requestManagement) pickFlowSchema(r *http.Request, flowSchemas FlowSchemaSeq, priorityLevelStates map[string]*PriorityLevelState) *rmtypesv1a1.FlowSchema {
+func hashFlowID(fsName, fDistinguisher string) uint64 {
 	// TODO: implement
-	return nil
+	// TODO(aaron-prindle) verify implementation
+	return hash(fmt.Sprintf("%s,%s", fsName, fDistinguisher))
```

Comment:
Since `hash.Hash` has a `Write` method that can be invoked multiple times, we do not need to pay to construct a string here.